app/src/main/java/com/kickstarter/ui/adapters/projectcampaign/ViewElementAdapter.kt

```diff
@@ -240,6 +241,7 @@ class ViewElementAdapter(
         (holder as? AudioElementViewHolder)?.let {
             it.setIsRecyclable(false)
             it.pausePlayer()
+            it.clearDisposables()
```

Comment:
I was thinking more in something like: 
`
(holder as? KSViewHolder)?.let {
            it.destroy()
        }
`
and make `destroy` not protected, in that way it can be applied to all viewHolders not just the AudioElementViewHolder, (at some point we will need to remove from `KSViewHolder` all references to trello's rxlifecycle library)

Reply:
Ok, I can do that too. One sec.